### PR TITLE
saml2aws: fix go_import_path.

### PR DESCRIPTION
**Pull request by ericonr**
@ScOut3R 

#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Review by ScOut3R — APPROVED** [review]
Thank you @ericonr, I think this was preventing me from updating saml2aws. :)

**Comment by ericonr** [comment]
@ScOut3R it seems they hadn't changed the module path in the version we ship... Can you take the change and include it in your PR to update the package? Then I will close this PR.

**Comment by ericonr** [comment]
@ScOut3R I've switched the PR to be an update, please test it out :)

File: srcpkgs/saml2aws/template [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Template file for 'saml2aws'
 pkgname=saml2aws
-version=2.26.2
+version=2.28.2
 revision=1
 build_style=go
-go_import_path=github.com/Versent/saml2aws
+go_import_path=github.com/Versent/saml2aws/v2
```
Comment (ScOut3R) [inline srcpkgs/saml2aws/template:6]: Thanks @ericonr!

I added the following small change and it works for me. Thank you for helping out with this update!

```suggestion
go_import_path=github.com/versent/saml2aws/v2
```

Comment (ericonr) [reply srcpkgs/saml2aws/template]: Dang it, thanks for catching!

**Pull request merged**